torch/autograd/function.py

```diff
@@ -115,8 +116,8 @@ def __init__(cls, name, bases, attrs):
 
         return super(FunctionMeta, cls).__init__(name, bases, attrs)
 
-
-class Function(with_metaclass(FunctionMeta, _C._FunctionBase, _ContextMethodMixin, _HookMixin)):
+# mypy doesn't understand `with_metaclass` from torch._six
```

Comment:
I wonder if we still need it. Might be worth testing

Reply:
It might work since the code from `[torch._six](https://github.com/pytorch/pytorch/blob/master/torch/_six.py)` is practically the same as python `[six](https://github.com/benjaminp/six/blob/master/six.py)` package. I can send another PR addressing this.